lib/compat/wordpress-5.9/class-wp-theme-json-5-9.php

```diff
@@ -229,6 +229,7 @@ class WP_Theme_JSON_5_9 {
 			'custom'           => null,
 			'customDuotone'    => null,
 			'customGradient'   => null,
+			'defaultDuotone'   => null,
```

Comment:
should it be defaultDuotones to match the syntax of defaultGradients ? 
There is only one defaultPalette , hence it's singular.

Reply:
Both the singular and plural seem natural to me. It's a palette of duotone filters. Just like `defaultPalette` is a palette of colors and `defaultGradients` is a palette of gradients.

I had it as singular to match `dutone` and `customDuotone` so all of the duotone settings are consistent with one another.